client/src/components/Workbench/Workbench.jsx

```diff
@@ -17,6 +17,7 @@ import * as ROUTES from '../../constants/routes';
 import NotificationToast from '../Toasts';
 import OnboardingAdmin from '../Onboarding/OnboardingAdmin';
 import './Workbench.scss';
+import '../../index.css'
```

Comment:
```suggestion
import '../../index.css' // avoid https://github.com/vitejs/vite/issues/3924
```

Just to see `index.css` loaded by `AnythingElse.js` instead of `index.js` is weird enough to give pause. How about a small comment.